### PR TITLE
Initial changes for javaee/j2ee -> jakartaee 

### PR DESCRIPTION
**Pull request by alwin-joseph**

### CODE REVIEW

**Comment by ankitoracle** [comment]
looks good.we should monitor the tests to fix if anything breaks

**Comment by scottmarlow** [comment]
Would it help reviewers, if we introduce a manual Jenkins job that builds a particular pull request or even combines a set of pull requests and produces something like a http://download.eclipse.org/ee4j/jakartaee-tck/8.0.1/pullrequestpreview/jakartaeetck.zip that could then be used to verify that the pull request doesn't break anything (including giving others a change to also verify if the pullrequestpreview/jakartaeetck.zip works for them).  If yes, I'll try hacking on the Jenkins jobs.  I suppose that jakartaee-tck-dev@eclipse.org could be a good place to discuss this as well. :)

File: release/tools/build.xml [hunk 1]
```diff
@@ -92,7 +92,7 @@
         <echo message=" jsf           [overrides]"/>
         <echo message=" jstl          [overrides]"/>
         <echo message=" el            [overrides]"/>
-        <echo message=" j2ee          [overrides]"/>
+        <echo message=" jakartae      [overrides]"/>
```
Comment (scottmarlow) [inline release/tools/build.xml]: Should this echo jakartaee or jakartaee?

**Comment by scottmarlow** [comment]
Other than the minor "jakartae" typo, changes look good to me also.

File: release/tools/build.xml (hunk 1 above)
Comment (alwin-joseph) [reply release/tools/build.xml]: Corrected this. Thanks

**Comment by alwin-joseph** [comment]
"Would it help reviewers, if we introduce a manual Jenkins job that builds a particular pull request or even combines a set of pull requests and produces something like a http://download.eclipse.org/ee4j/jakartaee-tck/8.0.1/pullrequestpreview/jakartaeetck.zip that could then be used to verify that the pull request doesn't break anything (including giving others a change to also verify if the pullrequestpreview/jakartaeetck.zip works for them). If yes, I'll try hacking on the Jenkins jobs. I suppose that jakartaee-tck-dev@eclipse.org could be a good place to discuss this as well. :)"

@scottmarlow 
1. Currently we are in the process of migrating our jenkins jobs to a new instance as the current one has stability issues. Hence I think we can wait till the runs are stable in the new instance to try out things. We will communicate about the new instance once they are available.
2. We already have a way to run the pull request version from the jenkins pipeline itself using "Scan Repository Now" option that runs the jobs on all the pending PRs in the respective branches. But I haven't experimented much on that and I feel it has limitations, for eg, it does not copy the tck bundle to any common location to be tested by someone else.
3. I believe a manual job would actually help to verify a pull request if not for this PR but for others. Glad to welcome your contribution on this.

**Pull request merged**